plugin/src/templates/utils.ts

```diff
@@ -43,6 +98,25 @@ export function prepareFilesystem(cacheDir: string): void {
   // Gatsby uses this instead of fs if present
   // eslint-disable-next-line no-underscore-dangle
   global._fsWrapper = lfs
+
+  console.log('Starting to prepare data directory')
+
+  if (process.env.LOAD_GATSBY_LMDB_DATASTORE_FROM_CDN === 'true') {
+    console.log('Starting to stream data file')
+
+    const dataMetadataPath = join(process.cwd(), 'public', 'dataMetadata.json')
+    const { fileName, url } = await readJSON(dataMetadataPath)
+    const downloadUrl = `${url}/${fileName}`
+
+    console.log('Downloading data file from', downloadUrl)
+
+    // Ensure the file to copy the downloaded file into exists
+    const filePath = join(TEMP_CACHE_DIR, 'data', 'datastore', 'data.mdb')
+    if (!existsSync(filePath)) {
+      ensureFileSync(filePath)
+    }
+    return downloadFile(downloadUrl, filePath)
+  }
   const dir = 'data'
```

Comment:
[pebble] I think the rest of this can be in an `else`, because we don't want to try copying stuff across in this case.

Reply:
I can write the `else` for clarity, but because line 18 is a `return` this chunk of code won't run when the block above  does